src/sentry/stacktraces.py

```diff
@@ -380,6 +380,16 @@ def get_stacktrace_processing_task(infos, processors):
     )
 
 
+def dedup_errors(errors):
+    # This operation scales bad but we do not expect that many items to
```

Comment:
I mean, this is fine, but why not just do like, `return list(set(errors))` ? Or are you intentionally trying to preserve the order?

Reply:
That and more importantly: errors are not hashable.

Reply:
🤔

Reply:
```pycon
>>> set([{'foo': 'bar'}])
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: unhashable type: 'dict'
```